src/main/resources/export/cpp/RobotContainer.h

```diff
@@ -0,0 +1,61 @@
+#set($command = $robot.getProperty("Autonomous Command").getValue())
+
+#header()
+
+#pragma once
+
+\#include "frc/Joystick.h"
+\#include "frc2/command/Command.h"
+
+\#include "Commands/#class($command).h"
+
+#foreach( $component in $components )
+#if ($component.getBase().getType() == "Command"
+     && $component.getProperty("Button on SmartDashboard").getValue())
+        #if( $component.getProperty("Parameter presets").getValue().isEmpty() )
+                #if ($component.getProperty("Requires").getValue() != "None")
+\#include "Subsystems/#class(${component.getProperty("Requires").getValue()}).h"
+               #end
+        #end
+#end
+#end
+
+#@autogenerated_code("prototypes", "	")
+#parse("${exporter_path}OI-includes.h")
+#end
+
+class RobotContainer {
+
+public:
+	RobotContainer();
+	frc2::Command* GetAutonomousCommand();
+
+#@autogenerated_code("prototypes", "	")
+#parse("${exporter_path}OI-prototypes.h")
+#end
+
+private:
+
+
+#foreach( $component in $components )
+#if ($component.getBase().getType() == "Command"
+     && $component.getProperty("Button on SmartDashboard").getValue())
+        #if( $component.getProperty("Parameter presets").getValue().isEmpty() )
+                #if ($component.getProperty("Requires").getValue() != "None")
+#class(${component.getProperty("Requires").getValue()})* m_#required_subsystem($component);
```

Comment:
If you have multiple commands that put buttons on smart dashboard that require the same subsystem, you'll get multiple instances of that subsystem. I think this can be simplified to just loop through and find all subsystems.

This also needs to be in an autogenerated block so that it gets updated when subsystems change